### ISSUE TITLE
`/local` and `/session` persists are conflicting in node environment

### ISSUE THREAD

**Issue by yumauri** [yumauri/effector-storage#26]
Check this code:

```javascript
import { persist as local } from 'effector-storage/local'
import { persist as session } from 'effector-storage/session'

const store1 = createStore(1)
const store2 = createStore(2)

local({ store: store1, key: 'store' })
session({ store: store2, key: 'store' })
```

In browser environment those two stores **will not be** synced, because they use different adapters
(even while they have same `key`).

But in node environment those stores magically uses same `nil` adapter, and thus stores **will be** synced.

This potentially could lead to some issues with SSR, when effector logic works in both worlds.

**Comment by igorkamyshev** [comment]
Should we add prefix to key in nil adapter?

**Comment by yumauri** [comment]
Not quite prefix to key, but I want to redo `nil` adapter, and add possibility to define [`keyArea`](https://github.com/yumauri/effector-storage#keyarea).

`nil` adapter is undocumented, and I doubt anyone uses it in a wild (like explicit adapter, I mean), so I wouldn't consider it's API change as breaking change, I think.

**Comment by yumauri** [comment]
Fixed in [version 4.4.0](https://github.com/yumauri/effector-storage/releases/tag/v4.4.0)
